README.md

```diff
@@ -1,2 +1,55 @@
 # telesto-base
-Base image for telesto.ai models
+Base image for telesto.ai models.
+
+# Instructions
+
+`telesto-base` contains a pip-installable Python package and a Docker image, allowing you to
+easily package your models for telesto.ai competitions.
+
+## telesto-base module
+To install the module, you can simply use pip:
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git
+```
+If you would like to use the latest not yet released version, you can install the one in the 
+`develop` branch.
+```
+pip install git+https://github.com/telesto-ai/telesto-base.git@develop
```

Comment:
I think these two pip commands are equivalent since we have `develop` as our default branch on Github.